### PR TITLE
Finish RHCOS product migration

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Removes ocp4 enablement from product rules
- Updates profiles to be in line with many of these changes
- Migrates CCEs to RHCOS as they were fully intended for RHCOS and not OCP

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Profile cis on ocp4:
 Newly added profile.
Profile e8 on ocp4:
 Rule audit_rules_login_events_lastlog, configure_crypto_policy, sshd_use_priv_separation, audit_rules_time_adjtimex, selinux_policytype, sysctl_kernel_dmesg_restrict, auditd_data_retention_flush, audit_rules_login_events, audit_rules_execution_seunshare, configure_ssh_crypto_policy, sysctl_kernel_kptr_restrict, sshd_set_loglevel_info, sysctl_kernel_unprivileged_bpf_disabled, audit_rules_time_clock_settime, audit_rules_execution_restorecon, audit_rules_time_stime, accounts_no_uid_except_zero, audit_rules_networkconfig_modification, audit_rules_time_settimeofday, audit_rules_dac_modification_chmod, audit_rules_execution_chcon, sshd_disable_root_login, audit_rules_login_events_tallylog, sshd_disable_rhosts, audit_rules_dac_modification_chown, selinux_state, audit_rules_execution_semanage, sysctl_kernel_randomize_va_space, audit_rules_time_watch_localtime, audit_rules_usergroup_modification, audit_rules_sysadmin_actions, auditd_log_format, sysctl_net_core_bpf_jit_harden, auditd_name_format, sshd_disable_gssapi_auth, service_auditd_enabled, audit_rules_kernel_module_loading, audit_rules_execution_setsebool, sshd_print_last_log, no_empty_passwords, audit_rules_execution_setfiles, sshd_disable_user_known_hosts, auditd_freq, sysctl_kernel_kexec_load_disabled, audit_rules_login_events_faillock, auditd_local_events, sshd_do_not_permit_user_env, sshd_enable_strictmodes, sshd_disable_empty_passwords, auditd_write_logs, sysctl_kernel_yama_ptrace_scope removed from e8 profile.
Profile moderate on ocp4:
 Rule audit_rules_unsuccessful_file_modification_open_o_creat, sysctl_net_ipv6_conf_all_accept_redirects, audit_rules_unsuccessful_file_modification_creat, kernel_module_freevxfs_disabled, kernel_module_can_disabled, audit_rules_unsuccessful_file_modification_lsetxattr, audit_rules_etc_passwd_openat, audit_rules_time_clock_settime, auditd_data_retention_admin_space_left_action, sysctl_net_ipv4_conf_default_secure_redirects, audit_rules_file_deletion_events_rmdir, audit_rules_unsuccessful_file_modification_removexattr, no_netrc_files, configure_kerberos_crypto_policy, audit_rules_unsuccessful_file_modification_open_o_trunc_write, wireless_disable_interfaces, audit_rules_execution_setsebool, audit_rules_dac_modification_lchown, grub2_vsyscall_argument, file_owner_sshd_config, service_debug-shell_disabled, grub2_audit_argument, service_usbguard_enabled, audit_rules_login_events_faillock, kernel_module_firewire-core_disabled, sysctl_net_ipv4_icmp_echo_ignore_broadcasts, audit_rules_file_deletion_events_rename, sysctl_net_ipv4_icmp_ignore_bogus_error_responses, audit_rules_dac_modification_fchmodat, chronyd_or_ntpd_specify_multiple_servers, kernel_module_tipc_disabled, file_ownership_var_log_audit, audit_rules_etc_passwd_open_by_handle_at, audit_rules_privileged_commands_gpasswd, file_permissions_var_log_audit, sysctl_net_ipv6_conf_default_accept_ra, kernel_module_cramfs_disabled, kernel_module_jffs2_disabled, configure_usbguard_auditbackend, audit_rules_unsuccessful_file_modification_open_by_handle_at_o_trunc_write, audit_rules_time_watch_localtime, sysctl_net_ipv6_conf_default_accept_redirects, audit_rules_unsuccessful_file_modification_openat, audit_rules_unsuccessful_file_modification_renameat, audit_rules_etc_gshadow_open, audit_rules_privileged_commands_sudo, audit_rules_etc_shadow_open, audit_rules_unsuccessful_file_modification_chown, audit_rules_unsuccessful_file_modification_open_by_handle_at_o_creat, chronyd_client_only, harden_ssh_client_crypto_policy, sysctl_net_ipv4_conf_all_log_martians, wireless_disable_in_bios, kernel_module_hfsplus_disabled, audit_rules_usergroup_modification_passwd, audit_rules_unsuccessful_file_modification_unlinkat, audit_rules_unsuccessful_file_modification_fchownat, harden_sshd_crypto_policy, directory_access_var_log_audit, sysctl_net_ipv4_conf_all_rp_filter, sysctl_kernel_kptr_restrict, audit_rules_dac_modification_fsetxattr, rpm_verify_ownership, audit_rules_etc_gshadow_openat, sshd_set_idle_timeout, audit_rules_dac_modification_fremovexattr, audit_rules_networkconfig_modification, audit_rules_privileged_commands_at, audit_rules_execution_chcon, account_disable_post_pw_expiration, kernel_module_sctp_disabled, auditd_data_retention_space_left, auditd_data_retention_action_mail_acct, enable_fips_mode, coredump_disable_storage, grub2_pti_argument, sysctl_net_ipv4_conf_default_rp_filter, file_permissions_sshd_pub_key, audit_rules_privileged_commands_usernetctl, audit_rules_etc_group_openat, audit_rules_privileged_commands_chage, audit_rules_dac_modification_removexattr, ensure_logrotate_activated, kernel_module_usb-storage_disabled, audit_rules_privileged_commands_mount, audit_rules_unsuccessful_file_modification_lremovexattr, no_tmux_in_shells, audit_rules_time_stime, audit_rules_file_deletion_events_unlink, audit_rules_time_settimeofday, no_shelllogin_for_systemaccounts, sysctl_net_ipv4_conf_default_send_redirects, package_audit_installed, audit_rules_privileged_commands_sudoedit, service_chronyd_or_ntpd_enabled, disable_users_coredumps, audit_rules_dac_modification_fchownat, audit_rules_sysadmin_actions, auditd_name_format, audit_rules_privileged_commands_passwd, auditd_data_retention_space_left_action, grub2_nousb_argument, audit_rules_unsuccessful_file_modification_ftruncate, audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order, grub2_disable_interactive_boot, audit_rules_privileged_commands_umount, sysctl_fs_protected_symlinks, audit_rules_execution_setfiles, audit_rules_unsuccessful_file_modification_openat_o_creat, audit_rules_usergroup_modification_group, chronyd_or_ntpd_specify_remote_server, kernel_module_atm_disabled, bios_disable_usb_boot, audit_rules_kernel_module_loading_finit, grub2_enable_selinux, sssd_run_as_sssd_user, configure_ssh_crypto_policy, audit_rules_execution_seunshare, audit_rules_unsuccessful_file_modification_setxattr, sysctl_kernel_unprivileged_bpf_disabled, accounts_no_uid_except_zero, audit_rules_unsuccessful_file_modification_rename, audit_rules_etc_gshadow_open_by_handle_at, audit_rules_privileged_commands_postqueue, audit_rules_usergroup_modification_shadow, grub2_audit_backlog_limit_argument, audit_rules_privileged_commands_unix_chkpwd, audit_rules_unsuccessful_file_modification_openat_rule_order, audit_rules_file_deletion_events_renameat, sysctl_net_ipv4_conf_default_accept_redirects, chronyd_no_chronyc_network, kernel_module_bluetooth_disabled, sysctl_net_ipv6_conf_all_accept_source_route, sysctl_net_ipv4_conf_all_accept_source_route, audit_rules_mac_modification, auditd_local_events, audit_rules_login_events_lastlog, usbguard_allow_hid_and_hub, selinux_confinement_of_daemons, auditd_data_disk_full_action, configure_crypto_policy, audit_rules_dac_modification_lsetxattr, audit_rules_unsuccessful_file_modification_open_by_handle_at, sysctl_fs_protected_hardlinks, require_singleuser_auth, selinux_policytype, disable_ctrlaltdel_burstaction, audit_rules_time_adjtimex, audit_rules_unsuccessful_file_modification_fchmodat, sysctl_net_ipv4_conf_default_accept_source_route, audit_rules_unsuccessful_file_modification_truncate, sysctl_net_ipv4_conf_all_send_redirects, audit_rules_unsuccessful_file_modification_fchown, audit_rules_unsuccessful_file_modification_open_rule_order, audit_rules_dac_modification_chmod, disable_ctrlaltdel_reboot, audit_rules_etc_shadow_open_by_handle_at, sysctl_net_ipv4_conf_all_secure_redirects, sysctl_net_ipv4_tcp_syncookies, audit_rules_privileged_commands_pam_timestamp_check, audit_rules_privileged_commands_newgrp, no_direct_root_logins, audit_rules_privileged_commands_pt_chown, audit_rules_execution_semanage, audit_rules_unsuccessful_file_modification_open, audit_rules_unsuccessful_file_modification_lchown, sysctl_net_ipv4_conf_all_accept_redirects, enable_dracut_fips_module, audit_rules_privileged_commands_chsh, sysctl_net_core_bpf_jit_harden, audit_rules_unsuccessful_file_modification_fremovexattr, service_auditd_enabled, banner_etc_issue, audit_rules_etc_passwd_open, no_empty_passwords, audit_rules_unsuccessful_file_modification_openat_o_trunc_write, sshd_set_keepalive, auditd_freq, auditd_write_logs, chronyd_or_ntpd_set_maxpoll, audit_rules_unsuccessful_file_modification_unlink, audit_rules_unsuccessful_file_modification_fchmod, directory_permissions_var_log_audit, audit_rules_dac_modification_setxattr, package_iptables_installed, sysctl_kernel_dmesg_restrict, file_groupowner_sshd_config, audit_rules_dac_modification_fchmod, audit_rules_kernel_module_loading_init, audit_rules_etc_group_open_by_handle_at, kernel_module_udf_disabled, auditd_data_disk_error_action, audit_rules_dac_modification_chown, coredump_disable_backtraces, service_bluetooth_disabled, network_nmcli_permissions, auditd_data_retention_max_log_file, auditd_log_format, audit_rules_privileged_commands_su, audit_rules_privileged_commands_ssh_keysign, sysctl_net_ipv6_conf_all_accept_ra, audit_rules_dac_modification_fchown, audit_rules_etc_shadow_openat, sysctl_net_ipv6_conf_default_accept_source_route, audit_rules_usergroup_modification_gshadow, audit_rules_privileged_commands_userhelper, file_permissions_sshd_private_key, audit_rules_privileged_commands_newuidmap, audit_rules_etc_group_open, service_rngd_enabled, sysctl_net_ipv4_conf_default_log_martians, auditd_data_retention_flush, audit_rules_media_export, grub2_uefi_password, audit_rules_immutable, audit_rules_dac_modification_lremovexattr, package_usbguard_installed, audit_rules_file_deletion_events_unlinkat, audit_rules_execution_restorecon, audit_rules_privileged_commands_postdrop, audit_rules_session_events, grub2_page_poison_argument, audit_rules_privileged_commands, audit_rules_unsuccessful_file_modification_fsetxattr, file_permissions_sshd_config, audit_rules_kernel_module_loading_delete, sshd_limit_user_access, audit_rules_unsuccessful_file_modification_chmod, sshd_disable_rhosts, audit_rules_login_events_tallylog, auditd_data_retention_max_log_file_action, auditd_data_retention_num_logs, kernel_module_squashfs_disabled, audit_rules_privileged_commands_newgidmap, selinux_state, audit_rules_usergroup_modification_opasswd, service_autofs_disabled, sysctl_kernel_perf_event_paranoid, service_systemd-coredump_disabled, sysctl_kernel_core_pattern, package_sudo_installed, rpm_verify_permissions, kernel_module_hfs_disabled, kernel_module_vfat_disabled, sysctl_kernel_kexec_load_disabled, audit_rules_privileged_commands_crontab, configure_openssl_crypto_policy, sysctl_kernel_yama_ptrace_scope removed from moderate profile.
Profile ncp on ocp4:
 Rule ocp_idp_no_htpasswd removed from ncp profile.

**Recommended tests to execute:**
 build_product ocp4
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml cis
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml e8
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml moderate
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml ncp

File: ocp4/profiles/cis.profile [hunk 1]
```diff
@@ -0,0 +1,105 @@
+documentation_complete: true
```
Comment (JAORMX) [inline ocp4/profiles/cis.profile:1]: Let's not add this profile just yet. It's using OCP3 only checks and won't work in OCP4.

**Comment by JAORMX** [comment]
Thanks for making these changes! Just one comment for the profile you added for ocp4.

**Comment by mildas** [comment]
**Changes identified:**
Profile e8 on ocp4:
 Rule ocp_idp_no_htpasswd added to e8 profile.
 Rule sysctl_kernel_kptr_restrict, audit_rules_login_events_faillock, audit_rules_execution_restorecon, audit_rules_login_events, accounts_no_uid_except_zero, sysctl_net_core_bpf_jit_harden, audit_rules_execution_setsebool, sysctl_kernel_kexec_load_disabled, sshd_disable_root_login, audit_rules_time_settimeofday, audit_rules_time_watch_localtime, auditd_local_events, audit_rules_dac_modification_chmod, auditd_name_format, sysctl_kernel_yama_ptrace_scope, audit_rules_time_clock_settime, audit_rules_kernel_module_loading, auditd_data_retention_flush, sshd_use_priv_separation, audit_rules_login_events_tallylog, selinux_state, audit_rules_execution_chcon, sshd_print_last_log, sshd_set_loglevel_info, audit_rules_sysadmin_actions, auditd_freq, configure_crypto_policy, audit_rules_networkconfig_modification, auditd_log_format, audit_rules_login_events_lastlog, service_auditd_enabled, sysctl_kernel_dmesg_restrict, audit_rules_execution_seunshare, sshd_disable_empty_passwords, audit_rules_time_stime, sshd_disable_user_known_hosts, no_empty_passwords, sysctl_kernel_randomize_va_space, audit_rules_execution_setfiles, sshd_do_not_permit_user_env, selinux_policytype, audit_rules_execution_semanage, sysctl_kernel_unprivileged_bpf_disabled, audit_rules_usergroup_modification, sshd_disable_gssapi_auth, configure_ssh_crypto_policy, sshd_disable_rhosts, sshd_enable_strictmodes, audit_rules_time_adjtimex, audit_rules_dac_modification_chown, auditd_write_logs removed from e8 profile.
Profile moderate on ocp4:
 Rule sysctl_kernel_kptr_restrict, audit_rules_privileged_commands, audit_rules_privileged_commands_unix_chkpwd, audit_rules_etc_group_open, audit_rules_dac_modification_setxattr, audit_rules_etc_group_openat, audit_rules_file_deletion_events_unlinkat, sysctl_net_ipv4_conf_all_rp_filter, grub2_disable_interactive_boot, audit_rules_privileged_commands_umount, auditd_data_retention_space_left, auditd_data_disk_full_action, auditd_data_retention_max_log_file, sysctl_net_ipv4_icmp_echo_ignore_broadcasts, audit_rules_privileged_commands_postqueue, sysctl_net_ipv4_conf_all_accept_source_route, audit_rules_immutable, kernel_module_hfsplus_disabled, kernel_module_tipc_disabled, sysctl_net_ipv4_conf_default_accept_redirects, harden_ssh_client_crypto_policy, auditd_freq, configure_usbguard_auditbackend, sshd_limit_user_access, audit_rules_dac_modification_lsetxattr, audit_rules_dac_modification_removexattr, configure_openssl_crypto_policy, audit_rules_unsuccessful_file_modification_fchown, audit_rules_execution_semanage, grub2_page_poison_argument, sysctl_kernel_unprivileged_bpf_disabled, audit_rules_privileged_commands_at, audit_rules_privileged_commands_pt_chown, sysctl_net_ipv4_conf_all_send_redirects, sysctl_net_ipv4_conf_default_send_redirects, service_chronyd_or_ntpd_enabled, grub2_audit_backlog_limit_argument, file_permissions_sshd_config, auditd_data_retention_max_log_file_action, sysctl_net_ipv4_tcp_syncookies, auditd_data_retention_admin_space_left_action, file_permissions_sshd_private_key, audit_rules_unsuccessful_file_modification_chmod, audit_rules_dac_modification_fremovexattr, audit_rules_kernel_module_loading_finit, auditd_data_retention_flush, directory_permissions_var_log_audit, sshd_set_keepalive, audit_rules_kernel_module_loading_init, rpm_verify_ownership, no_shelllogin_for_systemaccounts, enable_fips_mode, kernel_module_cramfs_disabled, audit_rules_etc_group_open_by_handle_at, audit_rules_dac_modification_lremovexattr, audit_rules_unsuccessful_file_modification_truncate, audit_rules_etc_shadow_open, kernel_module_vfat_disabled, auditd_data_retention_num_logs, audit_rules_privileged_commands_gpasswd, directory_access_var_log_audit, audit_rules_execution_setfiles, audit_rules_unsuccessful_file_modification_openat_o_trunc_write, kernel_module_usb-storage_disabled, configure_ssh_crypto_policy, sysctl_net_ipv4_conf_all_secure_redirects, audit_rules_dac_modification_chown, audit_rules_execution_restorecon, audit_rules_unsuccessful_file_modification_open, sysctl_net_core_bpf_jit_harden, audit_rules_privileged_commands_sudoedit, wireless_disable_in_bios, auditd_name_format, bios_disable_usb_boot, chronyd_no_chronyc_network, sysctl_fs_protected_hardlinks, require_singleuser_auth, grub2_uefi_password, disable_users_coredumps, audit_rules_privileged_commands_usernetctl, kernel_module_bluetooth_disabled, audit_rules_sysadmin_actions, grub2_audit_argument, service_systemd-coredump_disabled, chronyd_or_ntpd_set_maxpoll, audit_rules_time_stime, no_empty_passwords, kernel_module_freevxfs_disabled, chronyd_or_ntpd_specify_remote_server, service_usbguard_enabled, audit_rules_etc_gshadow_openat, harden_sshd_crypto_policy, sysctl_kernel_perf_event_paranoid, service_autofs_disabled, grub2_vsyscall_argument, audit_rules_unsuccessful_file_modification_lremovexattr, sysctl_net_ipv4_icmp_ignore_bogus_error_responses, grub2_pti_argument, audit_rules_dac_modification_fchmod, audit_rules_time_settimeofday, audit_rules_unsuccessful_file_modification_fsetxattr, grub2_enable_selinux, audit_rules_usergroup_modification_group, audit_rules_unsuccessful_file_modification_fchmod, audit_rules_unsuccessful_file_modification_rename, configure_kerberos_crypto_policy, audit_rules_file_deletion_events_renameat, selinux_confinement_of_daemons, audit_rules_login_events_tallylog, audit_rules_unsuccessful_file_modification_fchownat, ensure_logrotate_activated, audit_rules_file_deletion_events_unlink, audit_rules_unsuccessful_file_modification_setxattr, audit_rules_unsuccessful_file_modification_unlinkat, audit_rules_etc_passwd_open_by_handle_at, sssd_run_as_sssd_user, package_usbguard_installed, kernel_module_atm_disabled, configure_crypto_policy, audit_rules_etc_gshadow_open, service_auditd_enabled, sysctl_net_ipv6_conf_all_accept_redirects, audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order, service_rngd_enabled, audit_rules_unsuccessful_file_modification_renameat, sshd_set_idle_timeout, audit_rules_unsuccessful_file_modification_open_by_handle_at_o_creat, kernel_module_can_disabled, banner_etc_issue, audit_rules_etc_gshadow_open_by_handle_at, sysctl_fs_protected_symlinks, audit_rules_etc_passwd_open, audit_rules_dac_modification_lchown, service_debug-shell_disabled, audit_rules_privileged_commands_newgrp, audit_rules_unsuccessful_file_modification_chown, audit_rules_privileged_commands_chsh, audit_rules_unsuccessful_file_modification_unlink, audit_rules_execution_setsebool, audit_rules_dac_modification_fchmodat, audit_rules_unsuccessful_file_modification_open_o_creat, audit_rules_dac_modification_chmod, audit_rules_time_clock_settime, audit_rules_unsuccessful_file_modification_lchown, audit_rules_dac_modification_fchownat, no_netrc_files, enable_dracut_fips_module, audit_rules_privileged_commands_newgidmap, audit_rules_privileged_commands_ssh_keysign, audit_rules_unsuccessful_file_modification_lsetxattr, audit_rules_file_deletion_events_rename, audit_rules_unsuccessful_file_modification_removexattr, sysctl_net_ipv4_conf_default_accept_source_route, audit_rules_networkconfig_modification, disable_ctrlaltdel_burstaction, package_sudo_installed, auditd_data_retention_action_mail_acct, chronyd_or_ntpd_specify_multiple_servers, file_permissions_var_log_audit, sysctl_kernel_dmesg_restrict, disable_ctrlaltdel_reboot, selinux_policytype, package_iptables_installed, audit_rules_unsuccessful_file_modification_open_o_trunc_write, no_tmux_in_shells, audit_rules_unsuccessful_file_modification_ftruncate, auditd_data_disk_error_action, audit_rules_unsuccessful_file_modification_openat, audit_rules_privileged_commands_chage, sysctl_net_ipv6_conf_default_accept_ra, file_groupowner_sshd_config, sysctl_kernel_core_pattern, rpm_verify_permissions, audit_rules_privileged_commands_passwd, coredump_disable_storage, audit_rules_time_watch_localtime, auditd_local_events, audit_rules_unsuccessful_file_modification_creat, audit_rules_unsuccessful_file_modification_openat_o_creat, sysctl_net_ipv4_conf_default_rp_filter, file_owner_sshd_config, sysctl_net_ipv4_conf_default_log_martians, audit_rules_execution_chcon, sysctl_net_ipv4_conf_default_secure_redirects, audit_rules_privileged_commands_pam_timestamp_check, network_nmcli_permissions, audit_rules_kernel_module_loading_delete, sysctl_net_ipv4_conf_all_accept_redirects, grub2_nousb_argument, audit_rules_unsuccessful_file_modification_fremovexattr, auditd_log_format, chronyd_client_only, audit_rules_usergroup_modification_shadow, audit_rules_dac_modification_fsetxattr, coredump_disable_backtraces, kernel_module_udf_disabled, sysctl_net_ipv4_conf_all_log_martians, auditd_write_logs, sysctl_net_ipv6_conf_all_accept_ra, audit_rules_login_events_faillock, kernel_module_sctp_disabled, audit_rules_file_deletion_events_rmdir, sysctl_kernel_kexec_load_disabled, wireless_disable_interfaces, account_disable_post_pw_expiration, audit_rules_usergroup_modification_opasswd, no_direct_root_logins, kernel_module_jffs2_disabled, sysctl_net_ipv6_conf_default_accept_source_route, sysctl_net_ipv6_conf_all_accept_source_route, selinux_state, audit_rules_mac_modification, audit_rules_unsuccessful_file_modification_openat_rule_order, usbguard_allow_hid_and_hub, audit_rules_unsuccessful_file_modification_fchmodat, audit_rules_login_events_lastlog, audit_rules_dac_modification_fchown, audit_rules_execution_seunshare, audit_rules_etc_shadow_open_by_handle_at, kernel_module_firewire-core_disabled, audit_rules_privileged_commands_userhelper, audit_rules_etc_passwd_openat, audit_rules_session_events, kernel_module_squashfs_disabled, audit_rules_privileged_commands_postdrop, accounts_no_uid_except_zero, file_ownership_var_log_audit, audit_rules_privileged_commands_su, sysctl_net_ipv6_conf_default_accept_redirects, audit_rules_media_export, sysctl_kernel_yama_ptrace_scope, audit_rules_privileged_commands_newuidmap, audit_rules_unsuccessful_file_modification_open_rule_order, audit_rules_privileged_commands_mount, package_audit_installed, audit_rules_unsuccessful_file_modification_open_by_handle_at_o_trunc_write, audit_rules_usergroup_modification_gshadow, audit_rules_unsuccessful_file_modification_open_by_handle_at, audit_rules_privileged_commands_crontab, file_permissions_sshd_pub_key, auditd_data_retention_space_left_action, kernel_module_hfs_disabled, service_bluetooth_disabled, audit_rules_usergroup_modification_passwd, audit_rules_etc_shadow_openat, sshd_disable_rhosts, audit_rules_time_adjtimex, audit_rules_privileged_commands_sudo removed from moderate profile.
 Rule ocp_idp_no_htpasswd added to moderate profile.

**Recommended tests to execute:**
 build_product ocp4
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml e8
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml moderate

File: ocp4/profiles/ncp.profile [hunk 2]
```diff
@@ -1,19 +1,24 @@
 documentation_complete: true
```
Comment (JAORMX) [inline ocp4/profiles/ncp.profile:1]: This is meant to be FedRAMP moderate. but API/platform specific checks. let's not change this specific profile

Comment (JAORMX) [reply ocp4/profiles/ncp.profile:1]: nevermind, GitHub just showed the diff weirdly

**Pull request merged**